lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
```

Comment:
Agreed. I'm not wedded to the ``view`` parameter name, so feel free to come up with something better. 

FYI, here's the User API end point that uses this parameter:

http://edx-platform-api.readthedocs.org/en/latest/user/accounts.html#get-and-update-the-user-s-account-information